docs/workshops/interviews/technical/workshop.md

```diff
@@ -0,0 +1,262 @@
+---
+id: workshop
+title: Technical Interviews
+sidebar_label: Technical Interviews
+---
+
+# Technical Interviews
+
+Companies use different kinds of interviews when working out who to hire. Not all companies require technical interviews when applying, but for those that do, the style of interview is quite different from other interviews. It's useful to be prepared for this kind of interview.
+
+## What is a technical interview?
+
+Companies use technical interviews to test your technical aptitude. Technical interviews focus on how you will approach problems in the future if you get the job. They tend to focus on a few areas:
+
+* How do you think about tasks and how to solve them?
+* How able are you to turn an idea for how to solve a problem into code?
+* How well can you communicate your thoughts?
+* Are you easy to help when you're stuck?
+* How open are you to other ideas?
+
+Most other interview types expect you to have answers to most of the questions you're asked. Technical interviews are _not_ there just to show that you know all of the answers. You are expected to ask questions, and it's fine if you don't know some things - that's part of the point of the interview, to see how you work when you _don't_ already know all of the answers.
+
+## What is your goal in a technical interview?
+
+Your goal in a technical interview is to show that you're going to be someone the interviewer wants to work with.
+
+Think about people you've enjoyed working with - what traits do they have? I'll suggest a few:
+
+* Friendly - no one likes someone getting angry with them!
+* Patient.
+* Thoughtful - jumping straight into your first idea doesn't make it likely that you're doing the best thing.
+* Open to feedback - if you have an idea that may help me, it's frustrating if I don't listen to it.
+* Communicative - if I'm stuck, and you don't know what I'm thinking, it's hard for you to participate.
+* Able to think through a problem.
+* Able to write code that works.
+
+Notice that **most of these are not technical traits**. While you're solving technical problems in a technical interview, most of the point of the interview is to see how you interact with people, and how you think.
```

Comment:
I wonder if there's some value in giving insight into the interviewer's point of view here, to reinforce the point.

We could note that most interviewers will have a checklist similar to the list above that they have to say "I saw evidence of X" / "I didn't see evidence of X", and therefore it's your job to make sure they can check things off the list.

Reply:
This comes up as a theme throughout, so an alternative suggestion would be to recap all the things mentioned at the end as a summary.